cmd/deployment/show.go

```diff
@@ -112,4 +123,5 @@ func initShowFlags() {
 	showCmd.Flags().BoolP("metadata", "m", false, "Shows the deployment metadata")
 	showCmd.Flags().BoolP("settings", "s", false, "Shows the deployment settings")
 	showCmd.Flags().Bool("generate-update-payload", false, "Outputs JSON which can be used as an argument for the --file flag with the update command.")
+	showCmd.Flags().Bool("clear-transient", false, "Removes the `transient` field in order to make read - edit - write loop safe")
```

Comment:
should the comment explain that its default is true when generate-update-payload is enabled?

Reply:
Yeah I think so. I updated documentation.